halyard-config/src/main/java/com/netflix/spinnaker/halyard/validate/v1/providers/ValidateFileExists.java

```diff
@@ -21,6 +21,9 @@
 import java.io.*;
 import java.util.stream.Stream;
 
+/**
+ * Ensure that the given file exists, and is readable.
```

Comment:
I noticed several 'types' of validator descriptions:
"Generic validator that accepts/rejects..."
"Simple regex to see if..."
"Ensure that the given file exists..."

Might want to pick one style and use it throughout (I prefer the last style).